sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -37,8 +37,7 @@ import org.apache.spark.sql.catalyst.rules.Rule
 import org.apache.spark.sql.catalyst.trees.TreeNodeTag
 import org.apache.spark.sql.catalyst.util.ArrayData
 import org.apache.spark.sql.connector.read.Scan
-import org.apache.spark.sql.execution._
-import org.apache.spark.sql.execution.ScalarSubquery
+import org.apache.spark.sql.execution.{ScalarSubquery, _}
```

Comment:
nit

```
import org.apache.spark.sql.execution._
```

Reply:
Done

> I really like the comments in GpuSubqueryBroadcastExec. They help to explain what is happening. But just to be sure that I understand everything. `GpuSubqueryBroadcastExec` takes columnar as input and returns rows. `GpuBroadcastExchangeExec` always returns columnar data so it can be reused with anything else that needs columnar data. When AQE is enabled Spark knows that the output is columnar or not so it can adjust without us having to do it, and all we have to do is to match the pattern to know if we can replace this or not.
> 
> If I got anything wrong please correct me. I would also like to see a few others like @andygrove review this too before it is merged in.

Yes